.github/workflows/cicd.yml

```diff
@@ -64,18 +64,6 @@ jobs:
       code_coverage: true
       gather_dumps: true
 
-  # Run the API tests in GitHub.
-  ebpf_client:
```

Comment:
Why are we deleting these tests?

Reply:
Because they are also run in the driver tests. So these are redundant now.

Reply:
these are redundant as a separate task since the ebpf_client test runs in driver tests task as well.